linux/power_operation_scripts/power_operation_scripts.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Description:
+#   This test case is used for checking soft power operations
+#   When VMware tools is not installed or not running in VM, this test case result is 'Blocked'.
```

Comment:
Please change `VMware tools` to `VMware Tools`, and change `soft power operations` to `power operation scripts`. And please do not use `soft power` in below tasks either, as it is an official description.